### PR TITLE
SSR1500 device map

### PR DESCRIPTION
**Pull request by MichaelBaj**
Device Mapping for SSR1500 platform.
Based on requirements from https://128technology.atlassian.net/browse/I95-47703

### CODE REVIEW

**Comment by MichaelBaj** [comment]
- [ ] Need to add bezel and site-specific deployment image

**Review by BenMatase — APPROVED** [review]

**Review by BenMatase — CHANGES_REQUESTED** [review]
wait for bezel

File: devicemaps.json [hunk 1]
```diff
@@ -1139,6 +1139,234 @@
...
+              "standaloneBranch": {
+                "name": "ha-sync",
+                "description": "HA sync network interface, connected to port 1/3 labeled on the device"
+              }
+            }
+          },
+          {
+            "pciAddress": "0000:42:00.0",
+            "type": "HAFabric",
+            "speed": 10000,
```
Comment (agrawalkaushik) [inline devicemaps.json:1215]: 10G for HAFabric? Is that intentional and does that line up with other similar platforms? Just asking coz it seemed a bit off.

**Review by agrawalkaushik — APPROVED** [review]
Just the one clarification on HAFabric from @reidstidolph and @avijitvjuniper

**Review by bkamble — APPROVED** [review]

File: devicemaps.json (hunk 1 above)
Comment (MichaelBaj) [reply devicemaps.json:1215]: LAN and WAN interfaces are also 10+Gbps, therefore HA fabric interface for dogleg traffic needs to be similarly sized.

**Review by bkamble — CHANGES_REQUESTED** [review]
xe-3-1 is lan5
AND
xe-3-2 is lan6

https://github.com/128technology/interfacemaps/pull/15/files

**Comment by MichaelBaj** [comment]
Corrected image for lan5 and lan6

**Review by agrawalkaushik — APPROVED** [review]

**Pull request merged**